unity-client/Assets/Scripts/MainScripts/DCL/Controllers/VoiceChat/DCLVoiceChatController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL
+{
+    public class DCLVoiceChatController : MonoBehaviour
```

Comment:
I think we can safely use a standard C# class here and initialize it from `SceneController.Awake()` .

Reply:
But wouldn't that make necessary to have the `voiceChatAction` as a reference in `SceneController`? Wouldn't that go against splitting `SceneController` into more cohesive chunks?